elements/head.html

```diff
@@ -0,0 +1,50 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        <!-- Actual Tag Name; Whatever the tag name is.  -->
+        &lt;tagname>
```

Comment:
Should this be `tagname` or `head`?

Is the reason you're using `&lt;` here so that it doesn't get interpreted as HTML and displays rather than trying to parse it as a head tag?